test/cmd/router.sh

```diff
@@ -78,11 +78,12 @@ os::cmd::expect_success "oc adm router -o yaml | oc delete -f -"
 echo "router: ok"
 
 # test ipfailover
-os::cmd::expect_failure_and_text 'oc adm ipfailover --dry-run' 'you must specify at least one virtual IP address'
+os::cmd::expect_failure_and_text 'oc adm ipfailover --dry-run' 'service account "ipfailover" does not have sufficient privileges'
```

Comment:
That's a concerning change

Reply:
Ah, the reason for this change was because before, we called the Generate method on the ipfailover plugin _before_ validating the service account.

After this update, we validate the service account _before_ we call the Generate method on the ipfailover plugin. The validation for the service account, now happens in the command's `Validate` method, which is called before the `Run` method. The `Generate` method is called inside of the Run method now.

Reply:
No behavior changes here, as service account validation used to fail before this update with the same error, until a security context was created for it.

Reply:
Thanks for detailed explanation.